styles/style.styl

```diff
@@ -1,9 +1,12 @@
 
-form 
-    padding: 20px;
-
 .mybutton
     margin: 10px;
 
 .uk-search-default
     width: 200px;
+
+.size-13
+    font-size: 13px;
```

Comment:
Stylus provides variables. To make the project consistent, I recommend creating one namespace with variables, importing it in the main `.styl` file and then setting the font-sizes using those variables.

Reply:
Fixed